spec/cookbook_spec.rb

```diff
@@ -216,7 +226,8 @@ def repo_path
         end
       end
       fixtures.each do |fixture|
-        it "should handle #{fixture[:name]}" do
+        next if fixture[:name].include?('symlink') && track_symlinks
```

Comment:
why? test both ways.

In fact, what I'd do here is if there is add an optional `result_with_symlinks_on` entry in your hash. then below do:

```
expected = fixture[:result]
if track_symlinks && fixture[:result_with_symlink_tracking]
  expected = fixture[:result_with_symlink_tracking]
end
```

and then do `... to eq(expected)`